src/components/Tooltip/index.tsx

```diff
@@ -105,7 +105,7 @@ const Tooltip = ({
         <Content
           side="top"
           sideOffset={2}
-          className="max-w-80 px-5 text-sm"
+          className="z-[10000] max-w-80 px-5 text-sm"
```

Comment:
Could we use one of the tokens?, in this case `z-popover`. Mainly to avoid z index conflicts with other components.